src/toast/util/toastTypes.ts

```diff
@@ -0,0 +1,16 @@
+interface ToastData {
+  mode: "danger" | "success";
+  component: React.ReactNode;
```

Comment:
I think `content` can be a more descriptive name for this, "content of the toast" makes more sense to me. What do you think?

Reply:
Sounds good. We can change it as `content`.